mysql-test/suite/galera/t/galera_sst_rsync_data_dir.test

```diff
@@ -13,5 +13,4 @@
 --source include/auto_increment_offset_restore.inc
 
 # cleanup temporary database files:
---remove_files_wildcard $MYSQL_TMP_DIR/rsync_test *
---rmdir $MYSQL_TMP_DIR/rsync_test
```

Comment:
Premature removal of the test directory results in a "technical" error detected by the mtr script because without data home directory it is impossible to write a temporary innodb file during server shutdown.